src/Sulu/Component/Content/Mapper/ContentMapper.php

```diff
@@ -725,22 +729,22 @@ private function copyOrMove($uuid, $destParentUuid, $userId, $webspaceKey, $loca
             $this->documentManager->refresh($parentDocument);
         }
 
+        $originalDocument = $document;
```

Comment:
Why not naming the upper `$document` `$originalDocument` immediately? Then we would not have to reassign something to `$document`.